recipes/openmm-dlext/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+rm -rf build || true
+
+CMAKE_FLAGS="  -DCMAKE_INSTALL_PREFIX=${PREFIX}"
+CMAKE_FLAGS+=" -DCMAKE_BUILD_TYPE=Release"
+
+CMAKE_FLAGS+=" -DOPENMM_DIR=${PREFIX}"
+
+# if CUDA_HOME is defined and not empty, we enable CUDA
+if [[ -n ${CUDA_HOME-} ]]; then
+    CMAKE_FLAGS+=" -DCUDA_TOOLKIT_ROOT_DIR=${CUDA_HOME}"
+    CMAKE_FLAGS+=" -DCMAKE_LIBRARY_PATH=${CUDA_HOME}/lib64/stubs"
+fi
+
+if [[ "$target_platform" == osx* ]]; then
+    CMAKE_FLAGS+=" -DCMAKE_OSX_SYSROOT=${CONDA_BUILD_SYSROOT}"
```

Comment:
```suggestion
    CMAKE_FLAGS+=" -DPython_ROOT_DIR=${PREFIX}"
    CMAKE_FLAGS+=" -DCMAKE_OSX_SYSROOT=${CONDA_BUILD_SYSROOT}"
```

In OSX cmake doesn't seem to find the system python instead of the conda one, so we will hint that we want the latter.